src/containers/App/App.test.js

```diff
@@ -1,5 +1,53 @@
+import React from 'react'
+import { mapStateToProps, mapDispatchToProps } from './App'
+import App from './App'
+import { shallow } from 'enzyme'
+
+
 describe('App', () => {
-  it.skip('should have one test', () => {
-    expect(false).toBe(true)
+  let wrapper
+  const mockProps = {
+    notes: [{id: 1, title: 'Note Title'}],
+    items: [],
+    isLoading: true,
+    error: '',
+    fetchNotes: jest.fn()
+  }
+  it('should match the correct snapshot', () => {
+    wrapper = shallow(<App {...mockProps} />)
```

Comment:
this is clean, nice job on separating out the mockProps to make the test more readable 👍